### ISSUE TITLE
Pre-condition Violation in QueryAtom.GetSmarts()

### ISSUE THREAD

**Issue by tduigou** [rdkit/rdkit#2450]
**Description:**

- RDKit Version: 2019.03.1
- Platform: Mac OS

The GetSmarts method of a QueryAtom object raises a "Pre-condition Violation":

```
>>> from rdkit.Chem import rdqueries
>>> qatom = rdqueries.AtomNumEqualsQueryAtom(6)
>>> qatom.GetSymbol()
'*'
>>> qatom.GetSmarts()
[10:50:05] 

****
Pre-condition Violation
no owner
Violation occurred on line 124 in file /Users/glandrum/anaconda5/conda-bld/rdkit_1554778814548/work/Code/GraphMol/Atom.h
Failed Expression: dp_mol
****

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: Pre-condition Violation
        no owner
        Violation occurred on line 124 in file Code/GraphMol/Atom.h
        Failed Expression: dp_mol
        RDKIT: 2019.03.1
        BOOST: 1_67

```

This piece of code did work with a previous version of RDKit (although I did not remember exactly which one).

As a workaround, it is possible to extract an "already created" QueryAtom from a RDKIt Mol created : 

```
>>> from rdkit.Chem import MolFromSmarts
>>> m = MolFromSmarts('[*]')
>>> qatom = m.GetAtomWithIdx(0)
>>> qatom.GetSymbol()
'*'
>>> qatom.GetSmarts()
'*'
```

Thanks!

**Comment by greglandrum** [comment]
Confirmed. Thanks for the report.

**Comment by greglandrum** [comment]
This started with the fix for #1756.
